sdk/tests/deqp/functional/gles3/es3fFboColorbufferTests.js

```diff
@@ -226,7 +226,6 @@ es3fFboColorbufferTests.FboColorClearCase.prototype.render = function(dst) {
             var color = es3fFboColorbufferTests.randomVector(rnd, fmtInfo.valueMin, fmtInfo.valueMax);
 
             ctx.scissor(x, y, w, h);
-            ctx.clearBufferfv(gl.COLOR, 0, color);
```

Comment:
c++ doesn't have this call: https://cs.chromium.org/chromium/src/third_party/deqp/src/modules/gles3/functional/es3fFboColorbufferTests.cpp?rcl=0&l=138